src/screens/AlertScreen.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from "react";
+import * as WebBrowser from "expo-web-browser";
+import {
+  Platform,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from "react-native";
+
+
+const styles = StyleSheet.create({
+  outer:{
+    flex:1,
+    backgroundColor: '#b20700',
+  },
+  titleContainer:{
+    flex: 0.5,
+    justifyContent: 'center',
+  },
+  textContainer: {
+    flex: 0.5,
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  buttonContainer:{
+    flex:1,
+    justifyContent:'center',
+    alignItems: 'center',
+  },
+  welcome: {
+    marginLeft: '10%',
+    textAlign: 'left',
+    fontSize: 50,
+    color: '#FFFFFF',
+  },
+  paragraph: {
+    marginVertical: "10%",
+    marginHorizontal: "10%",
+    fontSize: 18,
+    alignItems: 'center',
+    textAlign: 'left',
+    color: '#FFFFFF',
+  },
+  button: {
+    position: 'absolute',
+    width: "80%",
+    height:"3em",
+    borderRadius:10,
+    justifyContent:'center',
+    backgroundColor: '#FFFFFF',
+    bottom:25,
+  },
+  buttonText:{
+    textAlign:'center',
+
+  },
+});
+
+class AlertScreen extends Component{
+
+  state = {
+    alert: null,
+    errorMessage:null,
+  };
+
+  constructor(props) {
+    super(props);
+    this.watchAlert();
+
+  }
+
+  componentDidMount() {
+    this.watchAlert();
+  }
+
+  watchAlert = async () => {
+    let alert = await this.state.alert
+  }
+
+  render() {
+    let text = 'Any critical messages will update this text.. lets see how it looks when there is a lot of text and it needs to go to a new line or something like that \n \n Please report any symptoms that you might have.';
+    if (this.state.errorMessage) {
+      text = this.state.errorMessage;
+    } else if (this.state.alert) {
+      text = JSON.stringify(this.state.alert);
```

Comment:
I'd recommend removing all references to state and JSON. This will eventually come from the redux store, so a hardcoded string (which you have) is fine.